src/image/loading_displaying.js

```diff
@@ -223,7 +221,14 @@ p5.prototype.image =
   var pd = 1;
 
   if (img.elt && img.elt.videoWidth && img.elt.style.width && !img.canvas) {
-    pd = img.elt.videoWidth / parseInt(img.elt.style.width, 10);
+    //if img.elt.size() has been used and no width passed to image()
+    if(img.elt.style.width && !dWidth){
+      pd = img.elt.videoWidth / parseInt(img.elt.style.width, 10);
```

Comment:
to clean this a little, we can pull out the `parseInt(img.elt.style.width, 10);` part and do that division at the end. that is, reformat lines 223-235 to first check img.elt, then do the other checks inside that if structure, then doing the final division at the end.

```javascript
if (img.elt) {
  if (img.elt.videoWidth && ...
  } ...
  pd /= parseInt(img.elt.style.width, 10);
}